channels/microsoft365/accounts.rst

```diff
@@ -47,9 +47,8 @@ Accounts
 Email Notification
    This option is **not available on Microsoft365 channels**.
 
-   To configure the outgoing mail provider for system notifications,
-   use a :doc:`regular email notification channel </channels/email/accounts>`.
-   (Self-hosted installations only.)
+   (See the corresponding section on the
+   :doc:`Email Accounts </channels/email/accounts>` page to learn more.)
```

Comment:
Same as the google channel bit.

Reply:
Addressed in 0a03e47. Let me know what you think!